components/src/core/components/Button/__tests__/button.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Button.vue', () => {
   it('renders OXD button', () => {
     const label = 'Button';
     const wrapper = mount(Button, {
-      props: {label},
+      props: {label, type: 'button'},
```

Comment:
Can't we define the default type of button as `button`

Reply:
added in base component. can be overridden if needed.